src/watchdog.py

```diff
@@ -459,9 +460,9 @@ def calculate_suspiciousness(self, tower):
         tower.suspiciousness = 0
         # TODO: let's try some ML?
         self.logger.info(f"Calculating suspiciousness for {tower}")
-        us_centric = self.config.getboolean('general', 'run_us_centeric_heuristics')
-        if us_centric:
-            self.logger.warning(f"RUNNING US CENTRIC HEURISTICS; THIS WILL RESULT IN FALSE POSITIVES IF YOU ARE NOT IN THE US")
+        check_geo_codes = self.config.getboolean('general', 'check_geographic_codes')
+        if check_geo_codes:
+            self.logger.warning(f"CHECKING COUNTRY AND CARRIER CODES, IF YOU DIDN'T CONFIGURED THIS IN CONFIG.INI IT CAN LEAD TO FALSE POSITIVES. IN THAT CASE WE SUGGEST TO TURN check_geographic_codes TO false")
```

Comment:
I think we should move this warning so that it only happens once on startup not every time. and make more of it lowercase. But this shouldn't hold up the PR so I'm going to approve it.